src/shared/components/palette/case-flag/write-case-flag-field.component.spec.ts

```diff
@@ -165,7 +165,9 @@ describe('WriteCaseFlagFieldComponent', () => {
     expect(component.ngOnInit).toBeTruthy();
     expect(component.formGroup).toBeTruthy();
     expect(component.formGroup.validator).toBeTruthy();
+    if ( !component.isDisplayContextParameterUpdate ) {
```

Comment:
@tarikdogan-hmcts @waynepHMCTS - please fix formatting of `if` block (can remove spaces in `if` condition and indent Line 169)